pkg/monitor/cluster/clusterversions.go

```diff
@@ -27,9 +27,17 @@ func (mon *Monitor) emitClusterVersions(ctx context.Context) error {
 		}
 	}
 
+	var rpVersion string
+	if mon.oc.Properties.ProvisionedBy != "" {
+		rpVersion = mon.oc.Properties.ProvisionedBy
+	} else {
+		rpVersion = "unknown"
+	}
```

Comment:
don't think we need this - we'll only see "unknown" until we do a round of cluster upgrades

Reply:
Fair!